www/src/examples/Form/FormLabelSizing.js

```diff
@@ -1,4 +1,4 @@
-<Form.Group>
```

Comment:
hmm, are we still not wanting to use FormGroup here?

Reply:
Ah I kept this around from the previous changes for simplicity.  It just added some extra margins at the bottom of the example, which didn't do too much.